sycl/include/sycl/ext/oneapi/bindless_images_memory.hpp

```diff
@@ -39,6 +39,9 @@ class image_mem_impl {
                                const context &syclContext);
   __SYCL_EXPORT ~image_mem_impl();
 
+  image_mem_impl(const image_mem_impl&) = delete;
+  image_mem_impl& operator=(const image_mem_impl&) = delete;
```

Comment:
This class is managed by another class `image_mem`, which keeps this in a shared pointer. So it probably does not make sense to be able to copy.